app/models/user_option.rb

```diff
@@ -165,6 +176,7 @@ def update_tracked_topics
 #  theme_key                        :string
 #  theme_key_seq                    :integer          default(0), not null
 #  allow_private_messages           :boolean          default(TRUE), not null
+#  user_home                        :integer          default(0), not null
```

Comment:
this should not default to 0 it should default to nil, unset by user

Reply:
Done.